pkg/api/server/register_containers.go

```diff
@@ -661,11 +661,10 @@ func (s *APIServer) registerContainersHandlers(r *mux.Router) error {
 	//    type: boolean
 	//    description: Include namespace information
 	//    default: false
-	//  - in: query
 	//    name: pod
 	//    type: boolean
 	//    default: false
-	//    description: Include Pod ID and Name if applicable
+	//    description: Ignored. Previously included details on pod name and ID that are currently included by default.
```

Comment:
Is this a change that might mess up someone who's still expecting to find the Pod ID and name here?  I don't think so, but I'm not 100% sure.

Reply:
They will get the data whether they specify the option or not.